### PR TITLE
Docs: References - Ordinals API and TMS API 

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Relates to https://github.com/hirosystems/docs/issues/314. 

Deleted physical yaml files for Ordinals and Token metadata service and updated Docusaurus config file.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -280,11 +280,11 @@ module.exports = {
           },
           {
             route: '/ordinals',
-            spec: './docs/ordinals-openapi.yaml',
+            spec: 'https://ordinals-api.vercel.app/openapi.yaml',
           },
           {
             route: '/token-metadata-service',
```
Comment (rafaelcr) [inline docusaurus.config.js:286]: Do you also need to change this one to `/metadata`?

Comment (trichards-hiro) [reply docusaurus.config.js:286]: @rafaelcr I don't believe so. The two should be `/ordinals` and `/metadata`

Comment (rafaelcr) [reply docusaurus.config.js:286]: > The two should be /ordinals and /metadata

Agreed, but in this PR you updated the `/metadata` path in line 330 but forgot to update it here in line 286 as well (see how `/ordinals` do match in lines 326 and 282)

Comment (sabbyanandan) [reply docusaurus.config.js:286]: Sorry for the confusion, folks. The paths will be `/metadata` and `/ordinals`. Unfortunately, between this and https://github.com/hirosystems/docs/pull/318, there's discrepancies still. 

Better to close this in favor of https://github.com/hirosystems/docs/pull/318.

**Comment by sabbyanandan** [comment]
Closing based on the comment at: https://github.com/hirosystems/docs/pull/319#discussion_r1124895724.

**Pull request closed without merging**